lib/config.yml

```diff
@@ -9,6 +9,7 @@ web:
         ttl: 3600
     password:
       enabled: true
+      validationStrategy: stormpath
```

Comment:
For simple strings you don't have to use quotes in YAML, but for consistency I'll add quotes